routes/prototype/apply/steps.js

```diff
@@ -307,14 +307,14 @@ module.exports = {
         next: '/passport-options-overseas'
     },
     '/get-updates': {
-        next: '/passport-options',
-        forks: [{
-            target: '/relationship-applicant',
-            condition: function (req, res) {
-                return req.session['hmpo-wizard-common']['16-or-older'] == false &&
-                    req.session['hmpo-wizard-common']['rising-16'] == false;
-            }
-        }]
+        next: '/passport-options'
+        // forks: [{
+        // //     target: '/relationship-applicant',
+        // //     condition: function (req, res) {
+        // //         return req.session['hmpo-wizard-common']['16-or-older'] == false &&
+        // //             req.session['hmpo-wizard-common']['rising-16'] == false;
+        // //     }
+        //  }]
```

Comment:
Small thing but can we delete this please?

Reply:
Done.